config/pg_featureserv.toml.example

```diff
@@ -16,6 +16,10 @@ HttpPort = 9000
 # Note: do not add a trailing slash.
 #    UrlBase = "http://localhost:9000"
 
+# Optional path to add to the service base URL
+# If set, all routes will be prefixed with this path
```

Comment:
A couple of examples of non-default paths would be good.   Do they include a trailing slash?

Reply:
Added